sol/main.c

```diff
@@ -419,10 +417,7 @@ req_queue_init(struct sol_config *sol_conf)
 			mbits_to_bytes(sol_conf->req_channel_bw_mbps);
 	}
 
-
```

Comment:
I can't comment on line `if (sol_conf->req_channel_bw_mbps == 0) {` above, but could you write a new pull request to change the test to `<= 0`? This is a bug, so this new pull request can be quickly merged.

Reply:
Pull request #395 solves this issue.